docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
-- `sessionKey` is the session keypair public key.
-- `resourceAbilityRequests` is a lit of abilities that the session key is requesting to perform against the specified Lit resources during authentication. Read more [here](resources-and-abilities) about Lit Resources and Abilities.
-- `capabilities` is an array of one or more AuthSigs.
-- `issuedAt` is the time the SessionSig was issued.
-- `expiration` is the time the SessionSig becomes invalid.
-- `nodeAddress` is the specific URL the SessionSig is meant for.
+```javascript
+litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.DatilDev,
+    // This storageProvider object can be omitted if executing in a browser
+    storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+    },
+});
+```
+When a `storageProvider` is added to the `LitNodeClient` instance, all functions for generating session signatures will try to create a session keypair for you and store it in the local storage. 
 
-#### Capabilities
+The session keypair can also be generated with the `generateSessionKeyPair()` function. Doing this enables you to pass the generated session keypair as the optional `sessionKey` parameter when generating session signatures.
 
-The `capabilities` field is an array of one or more signatures. These capabilities authorize this AuthSig address to utilize the resources specified in the capabilities SIWE messages. These signatures would have the address from the top level AuthSig in their URI field. For example, notice the following in the AuthSig above:
+If an instance of `LocalStorage` is not provided as the `storageProvider`, a new session keypair will be generated each time the code runs, instead of reusing a single set of keys.
 
-```
-URI: lit:session:6a1f1e8a00b61867b85eaf329d6fdf855220ac3e32f44ec13e4db0dd303dea6a
-```
+### Resources you can Request
 
-#### Node Address
+You can pass an array of `resourceAbilityRequests` to any of the functions that generate session signatures. These will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
 
-The `nodeAddress` will be different for each node, which means that, for a 30-node network, the SDK will generate 30 different `sig` and `signedMessage` parameters.
+When session capability objects are omitted from functions generating session signatures, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. This should only be done when debugging, as allowing unspecified access control conditions is a security vulnerability. Read more [here](capability-objects) about how to create custom session capability objects.
```

Comment:
Consider:

When session capability objects are omitted from functions generating session signatures, the SDK defaults to creating a session capability object with maximum permissions (i.e. can use any Lit Resource with any Lit Ability that the signer of the `AuthSig` owns). This should only be done when debugging, as allowing unspecified access control conditions is a security vulnerability.

Read more [here](https://deploy-preview-311--lit-dev-docs.netlify.app/sdk/authentication/session-sigs/capability-objects) about how to create custom session capability objects.